app/views/layouts/application.html.erb

```diff
@@ -1,21 +1,50 @@
 <!DOCTYPE html>
```

Comment:
This code does not relate to this homework.

Reply:
delete it

Reply:
No, you didn't

Reply:
No, you didn't